console-csharp-snippets-sample/AuthenticationHelper.cs

```diff
@@ -143,11 +146,12 @@ public static async Task<string> GetTokenForAppAsync()
         /// <summary>
         /// Signs the user out of the service.
         /// </summary>
-        public static void SignOut()
+        public static async void SignOut()
         {
-            foreach (var user in IdentityClientApp.Users)
+            IEnumerable<IAccount> accounts = await IdentityClientApp.GetAccountsAsync();
```

Comment:
This will probably raise a `CollectionChangeException`. you might want to take

```CSharp
await identityClientApp.GetAccountsAsync().ToArray() 
```

in order to force all the iteration to take place before you act on removing elements from the collection

Reply:
Done.